adminservice/src/main/java/nu/yona/server/subscriptions/rest/WhitelistedNumberController.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2015, 2016 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.subscriptions.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import nu.yona.server.subscriptions.service.WhitelistedNumberService;
+
+@Controller
+@RequestMapping(value = "/whitelistedNumbers")
+public class WhitelistedNumberController
```

Comment:
White list and white-listed are spelled as two words (see [Merriam Webster](https://www.merriam-webster.com/dictionary/whitelist)). Given that this is temporary code, I'm OK leaving it in, except for user-facing texts.

What do you think?